### PR TITLE
feat(i18n): add translation example

### PR DESCRIPTION
**Pull request by Jasmine-liang**
Add translation examples covering all scenarios: text, text+variable, richtext

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,14 +28,18 @@
...
+    "i18n:compile": "formatjs compile-folder --ast lang compiled-lang",
+    "i18n": "npm run i18n:extract && npm run i18n:compile"
   },
   "dependencies": {
     "@apollo/react-common": "^3.1.3",
     "@apollo/react-hooks": "^3.1.3",
     "@apollo/react-ssr": "^3.1.3",
     "@artsy/fresnel": "^6.1.0",
     "@ensdomains/content-hash": "^2.5.7",
+    "@formatjs/cli": "^6.0.0",
```
Comment (robertu7) [inline package.json:42]: should be in `devDependencies`?

File: tsconfig.json [hunk 2]
```diff
@@ -4,9 +4,13 @@
     "allowSyntheticDefaultImports": true,
     "jsx": "preserve",
     "lib": [
+      "esnext.intl",
```
Comment (robertu7) [inline tsconfig.json:7]: any references on these changes?

File: src/views/Guide/index.tsx [hunk 3]
```diff
@@ -22,10 +24,16 @@ const Guide = () => {
...
         <section
           dangerouslySetInnerHTML={{
-            __html: translate({
-              ...content,
-              lang,
-            }),
+            // richtext translation
+            __html: intl.formatMessage(
+              {
+                defaultMessage: '{content}',
```
Comment (robertu7) [inline src/views/Guide/index.tsx:30]: IMO, `{content}` is confusing for users in Crowdin. As we discussed this afternoon, we may move this kind of long text to a single `lang` file. For now, we could skip this use case (keep `translate`).

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 2 above)
Comment (Jasmine-liang) [reply tsconfig.json:7]: Typescript usage in `react-intl`: https://formatjs.io/docs/react-intl/

**Pull request merged**